.github/workflows/publish-nuget.yml

```diff
@@ -73,17 +73,17 @@ jobs:
     timeout-minutes: ${{ inputs.timeout-minutes }}
     steps:
       - name: Checkout
-        uses: Lombiq/GitHub-Actions/.github/actions/checkout@dev
+        uses: Lombiq/GitHub-Actions/.github/actions/checkout@issue/OSOE-536
```

Comment:
You only need to change the branch selectors for actions that you've changed as part of your issue.